ui_feature.go

```diff
@@ -110,12 +161,11 @@ func (f *UIFeature) thePageShouldHaveTheFollowingContent(expectedAPIResponse *go
 
 	for selector, expectedContent := range contentElements {
 		var actualContent string
-		err = chromedp.Run(f.Chrome.ctx,
-			f.runWithTimeOut(&f.Chrome.ctx, f.waitTimeOut, chromedp.Tasks{
+		err = chromedp.Run(f.Chrome.Ctx,
+			f.RunWithTimeOut(&f.Chrome.Ctx, f.WaitTimeOut, chromedp.Tasks{
```

Comment:
If the RunWithTimeOut signature is changed then this will need to be changed too.